modules/DSS-05_branches.yml

```diff
@@ -6,10 +6,12 @@ screens:
       title: Using Branches
       image: github-branches.png
       presenter-script: |
-        Every day you will be editing files on your project or adding new files. You don't want to take risks with the code on your master branch so you need to create a branch.
-        
+        Branching is an essential part of using Git and GitHub. While other Version Control Systems can make branching really painful, in Git it's easy and necessary.
+
         When you create a branch, you are essentially creating an identical copy of the project at that point in time that is completely separate from the master branch. This keeps your the code on your master branch safe while you experiment and fix issues.
-        
+
+        Branches are used for creating new features, experimenting with new code, doing bug fixes, or really changing code in any way that you might want someone else to review.
```

Comment:
@1point618 Change to: 

"You will create a branch when you want to experiment with new features, fix bugs, or make any changes to your code."